build/rbac/gen-nfs-rbac.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -xeEuo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+pushd "$SCRIPT_DIR" &>/dev/stderr
+
+NFS_RBAC_YAML_FILE="$SCRIPT_DIR/../../deploy/examples/csi/nfs/required-rbac.yaml"
+
+tmpdir="$(mktemp -d)"
+WITHOUT_FILE="${tmpdir}"/without-nfs.yaml # intermediate file of yaml that doesn't include NFS RBAC
+WITH_FILE="${tmpdir}"/with-nfs.yaml # intermediate file of yaml that includes previous plus NFS RBAC
+
+./get-helm-rbac.sh > "$WITHOUT_FILE"
+
+export ADDITIONAL_HELM_CLI_OPTIONS="--set csi.nfs.enabled=true"
```

Comment:
NIce and flexible to join other files. It would be great to do the same thing for the monitoring/rbac.yaml in a future PR.